src/config/server.rs

```diff
@@ -51,6 +53,10 @@ impl Server {
     self.enable_query_validation.unwrap_or(true)
   }
 
+  pub fn get_hostname(&self) -> String {
+    self.hostname.clone().unwrap_or("0.0.0.0".to_string())
```

Comment:
```suggestion
    self.hostname.clone().unwrap_or("127.0.0.1".to_string())
```

Update jsonplaceholder.graphql with `0.0.0.0`